package.json

```diff
@@ -1,27 +1,106 @@
 {
-  "name": "bodhi-app",
-  "version": "0.2.0",
-  "description": "An open source blockchain explorer for monitoring Bodhi platform",
+  "name": "BodhiPM",
```

Comment:
Bodhi-App? PM for 'Prediction Market' isnt comprehensive

Reply:
How about `BodhiPredictionMarket`? Just think there's a possibility of maybe having a different app in the future.

Reply:
Just check with TT, let use`Bodhi`

Reply:
👍